apps/emqx/test/emqx_listeners_SUITE.erl

```diff
@@ -226,6 +226,48 @@ t_wss_conn(Config) ->
         ok = ssl:close(Socket)
     end).
 
+t_tls13_session_resumption(Config) ->
```

Comment:
maybe add another test to cover the resumed failed case and expecting the conn should still success with regular handshake?